src/main/kotlin/no/nav/familie/ef/iverksett/behandlingsstatistikk/BehandlingsstatistikkTask.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.familie.ef.iverksett.behandlingsstatistikk
+
+import no.nav.familie.ef.iverksett.featuretoggle.FeatureToggleService
+import no.nav.familie.ef.iverksett.iverksetting.IverksettingRepository
+import no.nav.familie.ef.iverksett.iverksetting.domene.IverksettOvergangsstønad
+import no.nav.familie.ef.iverksett.repository.findByIdOrThrow
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = BehandlingsstatistikkTask.TYPE,
+    beskrivelse = "Sender melding til behandlingsstatistikk om at behandling er G-omregnet",
+    settTilManuellOppfølgning = true,
+)
+class BehandlingsstatistikkTask(
+    val behandlingsstatistikkService: BehandlingsstatistikkService,
+    val behandlingsstatistikkProducer: BehandlingsstatistikkProducer,
+    val iverksettingRepository: IverksettingRepository,
+    val featureToggleService: FeatureToggleService,
+) : AsyncTaskStep {
+    override fun doTask(task: Task) {
+        val behandlingId = UUID.fromString(task.payload)
+        val iverksett = iverksettingRepository.findByIdOrThrow(behandlingId).data
+        if (iverksett is IverksettOvergangsstønad &&
+            iverksett.erGOmregning() &&
+            featureToggleService.isEnabled("familie.ef.sak.g-beregning-scheduler")
```

Comment:
Denne er kanskje litt skummel? Det gjør jo at den ferdigstiller tasken, men ikke har kjørt statistikk-delen?

Reply:
Tenker det er "safe" å fjerne featuretoggle herfra iallefall. 
Hvis vi featuretoggler så kan tasken feile hvis g-omregning og toggle er avskrudd?

Reply:
Tenkte det var en fin dobbelsikring, men i og for seg sant at den ikke behøver å være her da samme sjekk gjøres ved oppretting av task. Fjernet det 👍